indigo_app/templates/document/_annotations.html

```diff
@@ -2,25 +2,30 @@
 {% verbatim %}
 {{#unless permissions.readonly }}
   <div class="float-right">
-    <a href="#" class="dropdown-toggle btn btn-sm" data-toggle="dropdown"></a>
-    <div class="dropdown-menu">
-      {{#unless in_reply_to}}
-        {{#unless task}}
-          {{#if permissions.can_create_task}}
-            <a href="#" class="dropdown-item create-anntn-task">Create task</a>
+
+    {{#if id}}
+      <a href="#" class="dropdown-toggle btn btn-sm" data-toggle="dropdown"></a>
+    
```

Comment:
Pedantic: please remove this blank line, the dropdown toggle and its dropdown menu are logically one unit.